index.shtml new file (server side include) for webmasterNote

DG12 opened the pull request:
The addition of index.**s**html,

 with the include for webmasterNote.html,

 will allow ANY additional information to be added to the webpage without the need to change any other files (including css if written as self contained html)

Apache web server will prefer the server side include index.**s**html to index.html so removing index.html is not required (although it should be removed sometime to eliminate confusions well as the css code related to the now available message.)
The embedded message of "NOW AVAILABLE" has been removed.

DG12 committed:
display beaconID as integer

DG12 committed:
Add include file announcing available

 This file, webmasterNote.html is included in main page and 
    allows additional notes without modifying the main  page

DG12 committed:
start over from master

DG12 committed:
start index.html over from master

DG12 committed:
Merge branch 'master' into master

DG12 committed:
index.shtml server side include 

Apache will prefer the Shtml (server Side Include) to the index.html.

This implements the previously added webmasterNote.html allowing webmaster to add ANYTHING to the bottom of the page  without modifying the main web page or css files.
Removed previous lines for "now available" and added the include.
No other changes, not even to css (yet)

ojousima commented:
Please base your work on current master, this PR has deletions and then adds previously deleted lines. 
index.html works well for us as a page name, I'd recommend that anyone who needs the .shtml will rather create symlink on their installation.

DG12 committed:
Update index.shtml

DG12 committed:
Update index.shtml

DG12 committed:
replace 1.1MB background image with 44KB

Any reason that the background image bg.jpg is so big?
The one at  ruu.vi/images/bg.jpg now is 2000x1116 pixels and includes a lot of image that is not displayed.
See

https://www.webpagetest.org/result/171021_JP_7bfb4133edf283fe3cbe81e8dd9f1586/1/details/#step1_request7
(VERY cool test site)
which shows ,it taking 4.2 seconds!
This on is cropped to (720x1114) matched what is visible on the iPhone .
It is also available at http://mybeacons.info/images/bg.jpg
download is much faster.

DG12 committed:
replace 365KB -m image with 45KB one

Might as well use the same image for background

DG12 committed:
Layout but not yet functional

First line says "will be available", font color:gray .
These aren options I think  would be useful .
First implementation will be via cookies. Although they are not really permanent, no database is needed.
Comments are welcome.

DG12 committed:
Update config.html

DG12 committed:
version 1.8.0 -1 (get the 2byte tag MAC ready)

This version processes format 8.  '#' no longer used,  2 byte ID see MyBeacons.info/packetFormats
                ditch format 3 support it never came here anyway, ditch seperate style file,
                totally reformat display, ditch titles for measurments,
          hooks for todo  add configuration and their cookies and assorted additional stuff and history page

DG12 committed:
1.8.0 -1 info page

DG12 committed:
Update sw.js

Add 404,info,config

DG12 committed:
fix background

DG12 committed:
Add WYS (What You See) Tag ID format

This is the string you see from the widget

DG12 committed:
add host to see comment

DG12 commented:
Didn't want duplicate index.html.
